modules/cluster-logging-collector-tuning.adoc

```diff
@@ -6,6 +6,13 @@
 [id="cluster-logging-collector-tuning_{context}"]
 = Advanced configuration for the Fluentd log forwarder
 
+[IMPORTANT]
+====
+The following configuration options that are configured by using the `spec.forwarder.fluentd` stanza in the `ClusterLogging` custom resource have been deprecated. There is no equivalent configuration option in the `spec.collection` stanza, and these options are planned to be removed and no longer supported in a future version of the {logging} API.
```

Comment:
@abrennan89 I just retried this on Log 5.8 using `spec.collection` stanza. 
```
spec:
  collection:
    fluentd:
      buffer:
        chunkLimitSize: 8m
        flushInterval: 5s
        flushMode: interval
        flushThreadCount: 3
        overflowAction: throw_exception
        retryMaxInterval: 300s
        retryType: periodic
        retryWait: 1s
    type: fluentd
  logStore:
  ....
```
I do see the settings under fluentd.conf 
```
    <buffer>
      @type file
      path '/var/lib/fluentd/default'
      flush_mode interval
      flush_interval 5s
      flush_thread_count 3
      retry_type periodic
      retry_wait 1s
      retry_max_interval 300s
      retry_timeout 60m
      queued_chunks_limit_size "#{ENV['BUFFER_QUEUE_LIMIT'] || '32'}"
      total_limit_size "#{ENV['TOTAL_LIMIT_SIZE_PER_BUFFER'] || '8589934592'}"
      chunk_limit_size 8m
      overflow_action throw_exception
      disable_chunk_backup true
    </buffer>
```

I suggest we have 'Use the Vector collector to ensure that you have access to the latest, supported {logging} features and configuration options since fluentd will be deprecated and removed in a future version of the {logging} API' something along those lines :)